Update RESTful Provider timeout note (#126217)
Azure/azure-content: articles/active-directory-b2c/api-connectors-overview.md

![Diagram of a RESTful service claims exchange](media/api-connectors-overview/restful-service-claims-exchange.png)

> [!NOTE]

> > If there is slow or no response from the RESTful service to Azure AD B2C, the timeout is 30 seconds and the retry count is two times (meaning there are 3 tries in total). In April 2025 timeout durations and retries will be based on an execution policy with 10 second timeouts being the longest. Timeout duration and retries can be updated through tenant metadata settings.

Reviewer: ```suggestion
> If there is slow or no response from the RESTful service to Azure AD B2C, the timeout is 30 seconds and the retry count is two times (meaning there are 3 tries in total). Starting in **April 2025**, timeout durations and retries will be based on an execution policy, with the longest timeout being 10 seconds. You’ll be able to update timeout durations and retries through tenant setting.
```

> > If there is slow or no response from the RESTful service to Azure AD B2C, the timeout is 30 seconds and the retry count is two times (meaning there are 3 tries in total). Starting in **April 2025**, timeout durations and retries will be based on an execution policy, with the longest timeout being 10 seconds. You’ll be able to update timeout durations and retries through tenant setting.

Reviewer 2: @kengaderdus @skylock Can you re-word the change to remove the future product plan? The [Contributor Guide stipulates](https://review.learn.microsoft.com/en-us/help/platform/contribute-get-started-channel-guidance?branch=main#unauthorized-content) to "Only describe what is possible with the released product to avoid confusion or mistrust if future features are delayed or canceled."

Author: It is important to notify customers that there is a change coming. Is there an acceptable way to convey that?
